std_msgs/Header header
string skill_key
string status_json_encoded
